tools/clang/test/HLSLFileCheck/hlsl/payload_qualifier/access.hlsl

```diff
@@ -10,6 +10,18 @@
 // RUN: %dxc -T lib_6_6 main %s -enable-payload-qualifiers -D TEST_NUM=9 | FileCheck -check-prefix=CHK9 -input-file=stderr  %s
 // RUN: %dxc -T lib_6_6 main %s -enable-payload-qualifiers -D TEST_NUM=10 | FileCheck -check-prefix=CHK10 -input-file=stderr  %s
 // RUN: %dxc -T lib_6_6 main %s -enable-payload-qualifiers -D TEST_NUM=11 | FileCheck -check-prefix=CHK11 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=0 | FileCheck -check-prefix=CHK0 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=1 | FileCheck -check-prefix=CHK1 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=2 | FileCheck -check-prefix=CHK2 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=3 | FileCheck -check-prefix=CHK3 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=4 | FileCheck -check-prefix=CHK4 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=5 | FileCheck -check-prefix=CHK5 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=6 | FileCheck -check-prefix=CHK6 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=7 | FileCheck -check-prefix=CHK7 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=8 | FileCheck -check-prefix=CHK8 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=9 | FileCheck -check-prefix=CHK9 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=10 | FileCheck -check-prefix=CHK10 -input-file=stderr  %s
+// RUN: %dxc -T lib_6_7 main %s -D TEST_NUM=11 | FileCheck -check-prefix=CHK11 -input-file=stderr  %s
 
```

Comment:
Are there no lib_6_8 tests here because of #6462?

Reply:
No, this fixes #6462. The point where it makes a difference in treatment is 6.7. I could add 6.8 tests to make doubly sure, but I was trying to balance my usual propensity to test everything against the equally common pushback that I was using up test time with minimal benefit.